pkg/scheduler/plugins/proportion/proportion.go

```diff
@@ -235,18 +235,36 @@ func (pp *proportionPlugin) OnSessionOpen(ssn *framework.Session) {
 		return victims, util.Permit
 	})
 
-	ssn.AddOverusedFn(pp.Name(), func(obj interface{}) bool {
-		queue := obj.(*api.QueueInfo)
+	ssn.AddOverusedFn(pp.Name(), func(queue *api.QueueInfo) (bool, bool) {
```

Comment:
Whether this function can be simplied as below.
{
    ...
    isAllDimensionOverUsed := attr.deserved.LessEqual(attr.allocated, api.Zero)
    isOverUsed := !attr.allocated.LessEqual(attr.deserved, api.Zero)
    metrics.UpdateQueueOverused(attr.name, isOverUsed)
   ...
return isOverUsed, isAllDimensionOverUsed
}

Reply:
Good idea. Let me optimize it